src/lib/bootloader/grub2base.rb

```diff
@@ -54,6 +54,7 @@ def initialize
       @grub_default = ::CFA::Grub2::Default.new
       @sections = ::Bootloader::Sections.new
       @pmbr_action = :nothing
+      @console = nil
```

Comment:
this is not needed unless it is for some documentation purpose. By default all variables are nil.

Reply:
Yes, this is meant mostly as a reminder that the variable exists.